### ISSUE TITLE
Add sh1106 oled support

### ISSUE THREAD

**Issue by penfold42** [pi1541/Pi1541#28]
I will, when it arrives

**Comment by andremiller** [comment]
Which one are you planning on adding @penfold42 ? I see there's one with multiple interfaces (3-wire SPI, 4 wire SPI and I2C) and another that only has I2C.

**Comment by penfold42** [comment]
I2c

**Comment by penfold42** [comment]
Anyone out there got a sh1106 based display ?

**Comment by pluto63** [comment]
I do have the i2c 4 pin 1.3” oled, as for now it showed some garbage on startup then black. I’m using this type of oled on tzxduino and code needed to be modify for compatibility.

**Comment by penfold42** [comment]
Did you change the lcdname to sh1106_128x64

**Comment by pluto63** [comment]
Yes I did change it and it doesn’t work. Just some garbage for a second at startup.

**Comment by penfold42** [comment]
ok, thanks - I've ordered one so Ive done no real testing - just took a punt based on the datasheet

**Comment by penfold42** [comment]
#41
